frontend/src/utils/utils.ts

```diff
@@ -244,3 +244,18 @@ export function setEmptyStringsToNull(data: any) {
 
   return data;
 }
+
+/**
+ * @function toKebabCase
+ * Converts string values to kebab case
+ * @param  {string} input The object to change
+ * @returns {string} The object with the remapped values
+ */
+export function toKebabCase(input: string | undefined | '') {
```

Comment:
Typing: The specific type `''` is already a member of type `string` and doesn't need to be there. If you need to explicitly specify empty strings, that could be something the JSDoc field could mention in lieu.

Reply:
Fixed.